.aws/src/main.ts

```diff
@@ -318,6 +322,10 @@ class ListAPI extends TerraformStack {
               name: 'DATABASE_WRITE_PASSWORD',
               valueFrom: databaseSecretEnvVars.writePassword,
             },
+            {
+              name: 'UNLEASH_KEY',
+              valueFrom: `arn:aws:secretsmanager:${region.name}:${caller.accountId}:secret:${config.name}/${config.environment}/UNLEASH_KEY`,
```

Comment:
oo owe have keys!?

Reply:
Yep! I made them and put them in 1password (and AWS )